Packs/Sysdig/Integrations/Sysdig/Sysdig.yml

```diff
@@ -0,0 +1,256 @@
+      description: Max number of vulnerabilities to return.
+      name: limit
+    - auto: PREDEFINED
+      defaultValue: all
+      description: Type of workload to query for vulnerabilities
+      name: workloadType
+      predefined:
+      - all
+      - daemonset
+      - deployment
+      - statefulset
+      - pod
+      - replicaset
+    - auto: PREDEFINED
+      defaultValue: "no"
+      description: Return csv output
+      name: csv
+      predefined:
+      - "yes"
+      - "no"
+    description: List container vulnerabilities
+    name: sysdig-list-vulnerabilities
+    outputs:
+    - contextPath: Sysdig.ContainerVulns.policyEvaluationsResult
+      description: Policy Evaluations Result
+    - contextPath: Sysdig.ContainerVulns.exploitCount
+      description: Count of exploits
+      type: unknown
+    - contextPath: Sysdig.ContainerVulns.storedAt
+      description: Time stored
+      type: string
+    - contextPath: Sysdig.ContainerVulns.imageId
+      description: Image Id
+      type: string
+    - contextPath: Sysdig.ContainerVulns.resultId
+      description: Result Id
+      type: string
+    - contextPath: Sysdig.ContainerVulns.runningVulnsBySev
+      description: Running vulnerabilities by severity
+    - contextPath: Sysdig.ContainerVulns.vulnsBySev
+      description: Vulnerabilities by severity
+  - arguments:
+    - description: ID of container image to query (results from sysdig-list-policies)
+      name: resultId
+      required: true
+    - defaultValue: "20"
+      description: Number of results to return
+      name: limit
+    description: List individual vulnerabilities by container or pod
+    name: sysdig-list-vulns-by-container
+    outputs:
+    - contextPath: Sysdig.ContainerVulns.vuln_cvssScore
+      description: CVSS score
+      type: string
+    - contextPath: Sysdig.ContainerVulns.package_name
+      description: Name of vulnerable package
+      type: string
+    - contextPath: Sysdig.ContainerVulns.vuln_cvssVersion
+      description: CVSS version
+      type: string
+    - contextPath: Sysdig.ContainerVulns.vuln_exploitable
+      description: Is the vulnerability exploitable?
+      type: boolean
+    - contextPath: Sysdig.ContainerVulns.vuln_name
+      description: Name of vulnerability
+      type: string
+    - contextPath: Sysdig.ContainerVulns.package_type
+      description: Package type
+      type: string
+    - contextPath: Sysdig.ContainerVulns.package_id
+      description: Package Id
+      type: string
+    - contextPath: Sysdig.ContainerVulns.vuln_disclosureDate
+      description: Vulnerability disclosure date
+      type: date
+    - contextPath: Sysdig.ContainerVulns.id
+      description: Id
+      type: string
+    - contextPath: Sysdig.ContainerVulns.package_version
+      description: Package version
+      type: string
+    - contextPath: Sysdig.ContainerVulns.fixedInVersion
+      description: Fixed version
+      type: string
+  dockerimage: demisto/sysdig:1.0
```

Comment:
You need to create this docker image.
it should be easy procedure. Just follow the instructions here https://github.com/demisto/dockerfiles
create a PR and assign me, or add here the link to the PR and I will review